Zendesk destination documentation

mohan-h opened the pull request:
initial commit

mohan-h committed:
Zendesk destination initial commit

initial commit

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,158 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
```
In this authentication section, you are referring to the authentication steps in the reversed order. First you have to fill in the bearer token, and THEN you select Connect to destination. Please move the steps in the correct order.

mohan-h replied:
Hmm, sorry for the line transpose.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,158 @@
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
```
These look like required mappings. However, at a quick scanning of the article, readers might dismiss these as examples. I suggest creating a subheading here called **Required mappings**, and explain that these mappings must always be used for Zendesk.

mohan-h replied:
Thanks, rather than introducing a level 4 heading, have moved the [!IMPORTANT] block that followed the mappings table and mentioned the same.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,158 @@
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`xdm: person.name.lastName`|`Attribute: last_name`| Yes, Note you can also use `name`. |
```
The note in the Mandatory field looks very messy. I suggest updating the target field as such:
"`Attribute: last_name` or `Attribute: name`", and removing the note from the Mandatory field.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,158 @@
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** and navigate to the list of destinations.
+1. Next, select the destination and switch to the **[!UICONTROL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/crm/zendesk/destinations-activation-data.png)
```
Let's improve screenshots by eliminating anything that takes the attention away from the AEP UI. Please crop this screenshot so it doesn't show other browser toolbars and such.

mohan-h replied:
Apologies, thank you for noticing.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,158 @@
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count within the segment.
+![Platform UI screenshot example showing Segment.](../../assets/catalog/crm/zendesk/segment.png)
+
+1. Log in to the [!DNL Zendesk] website, then navigate to the **[!UICONTROL Contacts]** page to check if the profiles from the segment have been added. This list can be configured to display columns for the additional fields created with the segment Mapping ID and segment statuses.
```
Mapping ID should be enclosed in UICONTROL tags.

mohan-h replied:
Thanks

vfalon reviewed (changes requested):
Looks good! Just a few comments to address. Thanks!

mohan-h committed:
Review corrections

Corrections after review https://github.com/AdobeDocs/experience-platform.en/pull/153#pullrequestreview-1209958439

mohan-h committed:
Content rewording

mohan-h committed:
Content updates

mohan-h committed:
Content updates

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,166 @@
+---
+# [!DNL Zendesk] connection
+
+[!DNL Zendesk](https://www.zendesk.com) is a customer service solution and sales tool.
```
Missing brackets on this DNL link.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,166 @@
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `Bearer Token` | The Access Token you have generated in your [!DNL Zendesk] account. <br> Follow the documentation to [generate a [!DNL Zendesk] access token](https://developer.zendesk.com/documentation/sales-crm/first-call/#1-generate-an-access-token) if you do not have one.| `a0b1c2d3e4...v20w21x22y23z` |
```
"Bearer token" and "access token". No need to capitalize, as it's a common name.

vfalon commented on help/destinations/catalog/crm/zendesk.md:
```diff
@@ -0,0 +1,166 @@
+
+>[!IMPORTANT]
+>
+>Both the attribute mappings listed in the table which follows are mandatory and required for [!DNL Zendesk] to work.
```
This note is confusing because there is no table following it. Either move that part of the note next to the table, or rephrase as "Both attribute mappings listed in the table further below are mandatory and required for [!DNL Zendesk] to work".

mohan-h replied:
Changed the position of the note. 

Note @vfalon had also shared [comment](https://github.com/AdobeDocs/experience-platform.en/pull/153#discussion_r1043275221)

vfalon reviewed (changes requested):
Couple more comments. Otherwise looks good.

mohan-h committed:
Updated post review
